Automattic/jetpack
# AMP: Standard mode disables the WPCOM toolbar in wp-admin

Whilst developing #19525 we discovered that the WPCOM toolbar module was
disabled in the admin when the AMP template mode was set to standard.
This is because `amp_is_request` always returns `true`. 

#### Changes proposed in this Pull Request:
This change fixes the problem by adding an additional check for AMP being available, which
it is not in admin, CLI requests etc.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Install the AMP plugin
* Activate the WordPress.com toolbar option in Jetpack settings
* Set the template mode to Standard in the AMP settings
* Notice that the default toolbar is displayed in wp-admin
* With this branch, the WPCOM toolbar should be used in the admin, but will still be disabled on the front end of the site.
